### ISSUE TITLE
manhole gtk2: Gtk-ERROR **: Char offset 1 is off the end of the line
aborting...

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11392]
|<img alt="slyphon's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| slyphon reported|
|-|-|
|Trac ID|trac#484|
|Type|defect|
|Created|2004-01-30 20:35:12Z|

<details><summary>Searchable metadata</summary>

```
trac-id__484 484
type__defect defect
reporter__slyphon slyphon
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__worksforme worksforme
component__core core
keywords__ 
time__1075494912000000 1075494912000000
changetime__1470928013401775 1470928013401775
version__ 
owner__ 
cc__acapnotic cc__itamarst cc__slyphon
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="slyphon's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|slyphon commented|
|-|-|

```
#!html
<pre>
i get this when trying to execute a multiline function definition in manhole

  File "/home/slyphon/Projects/Twisted/twisted/manhole/ui/gtk2manhole.py", line
203, in _on_key_press_event
    buffer.place(point)
AttributeError: 'gtk.TextBuffer' object has no attribute 'place'
                                                                               
                                                                               
               
Gtk-ERROR **: Char offset 1 is off the end of the line
aborting...

I'm assigning this to itamar because i don't know who's responsible, and itamar
will know
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="keturn's avatar" src="https://avatars.githubusercontent.com/u/83819?s=50" width="50" height="50">](https://github.com/keturn)<a name="note_2"></a>|@keturn set status to `closed`|
|-|-|

Can't seem to reproduce this.

```
>>> def pants():
...     x = 1 + 1
...     return x
... 
>>> pants()
2
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset 095d01d0f3964179af94b5ba88df5fd463d6838d
```
#!CommitTicketReference repository="" revision="095d01d0f3964179af94b5ba88df5fd463d6838d"
Merge pull request #484 from twisted/8744-rodrigc-pair-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8744

Port twisted.pair to Python 3
```
